test/fixture/package.json

```diff
@@ -15,7 +15,7 @@
       "make_targets": {
         "win32": ["squirrel"],
         "darwin": ["zip"],
-        "linux": ["deb", "rpm"]
+        "linux": ["deb", "rpm", "flatpak"]
```

Comment:
This fixture file really doesn't matter and shouldn't really need to be changed most of the time 👍 

It could just be a JSON file with `config.forge` set to an empty object for it to work for the tests.